duckduckgo.safariextension/html/global.html

```diff
@@ -222,6 +222,10 @@
           }
 
           var query = evt.query;
+          if ((safari.extension.settings.ddg_locationbar=="bang") && !(/((^|\s)![^\s])|(^\s*\\)/.test(query))) {
```

Comment:
@rrotter if I am reading this correctly then this only matches the following situations:

- bang at the beginning of the query
- bang separated from other parts of the query by a whitespace
- a query with `\` in the beginning 


I am not sure if this is correct in terms of bang detection since DuckDuckGo would find a bang even in query `jedi!w` whereas this regular expression would not.

Was that intentional? Is there something I am missing here?